test/gtest/ucp/test_ucp_tag_xfer.cc

```diff
@@ -986,6 +986,36 @@ UCS_TEST_P(test_ucp_tag_stats, rndv_unexpected, "RNDV_THRESH=1000",
                       UCP_WORKER_STAT_TAG_RX_RNDV_UNEXP);
 }
 
+UCS_TEST_P(test_ucp_tag_xfer, generic_exp_mlane_eager,
+           "MAX_EAGER_LANES=3") {
+    test_xfer(&test_ucp_tag_xfer::test_xfer_generic, true, false, false);
+}
+
+UCS_TEST_P(test_ucp_tag_xfer, generic_exp_mlane_rndv,
```

Comment:
why just generic? maybe add some tests for contig, or just make all tests in this test suite use MAX_EAGER_LANES=2,MAX_RNDV_LANES=2? it's possible to set it from test::init() function